contracts/src/v0.8/KeeperRegistry1_2.sol

```diff
@@ -127,16 +127,6 @@ contract KeeperRegistry1_2 is
     uint32 nonce; // 2 evm words
   }
 
-  struct Upkeep {
```

Comment:
does `import {Upkeep as Upkeepv2} from "KeeperRegistry1_2"` work?

Reply:
it does not work bc this struct is defined privately in a contract.